build-image-src/Dockerfile-python27

```diff
@@ -56,6 +55,6 @@ ENV LANG=en_US.UTF-8
 # Python for it to be picked up during `sam build`
 RUN pip3 install wheel
 
-RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python get-pip.py && rm get-pip.py
```

Comment:
Has this been tested today? Making sure this isn't impacted by the recent `pip` deprecation (which frankly might be a broader issue for building Python 2.7 going forward beyond our scope).

Reply:
I've tested it builds, `pip` links to the Python 2 `pip`, and that a dummy `pip install` works fine.

Reply:
And for reference, that was added fairly recently: https://github.com/pypa/get-pip/pull/81

The homepage hasn't yet been updated, but refers to the same method for Python 2.6: https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py

Also https://stackoverflow.com/a/65866547